src/Surfnet/StepupMiddleware/CommandHandlingBundle/DependencyInjection/CompilerPass/AddPipelineStagesCompilerPass.php

```diff
@@ -18,19 +18,45 @@
 
 namespace Surfnet\StepupMiddleware\CommandHandlingBundle\DependencyInjection\CompilerPass;
 
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception\RuntimeException;
 use Symfony\Component\DependencyInjection\Reference;
 
 class AddPipelineStagesCompilerPass implements CompilerPassInterface
 {
+    /**
+     * {@inheritdoc} Since the priorities cannot be changed runtime but only through configuration, we're doing the
+     * sorting based on priority here. A lower priority means the stage is added earlier.
```

Comment:
This is just the way it already was configured to be honest. Don't mind swapping it around (small change). Yeah, I'll swap it around.